examples/README.md

```diff
@@ -0,0 +1,34 @@
+
+### Transistor Basics
+A transistor shall include either w or nfin parameter where w (nfin) refers to the width (the number of fins) in planar (finfet) process:
+```
+# A transistor in a planar process
+Mi0 d g s b pmos w=90n l=14nm m=4
+```
+```
+# A transistor in a finfet process
+Mi0 d g s b pmos nfin=4 l=14nm m=4
+```
+
+### Parallel Transistors
+An array of transistors in parallel can be specified using m and nf parameters. m specifies the number of instances. nf is optional and specifies the number of fingers for each instance.
+```
+# Four instances of single-finger transistors in a planar process (total of 4 fingers)
+Mi0 d g s b pmos w=90n l=14nm m=4
+```
+```
+# Four instances of two-fingered transistors in a planar process (total of 8 fingers)
+Mi0 d g s b pmos w=90n l=14nm m=4 nf=2
+```
+
+### Series (stacked) Transistors
```

Comment:
I am not sure about difference between m and stacked transistors.
Also, can we put images of generated cells in this readme?